alex/applications/PublicTransportInfoEN/data/database.py

```diff
@@ -10,57 +10,70 @@
 import re
 import sys
 
-
 from alex.utils.config import online_update, to_project_path
 
+
 __all__ = ['database']
 
 
 database = {
     "task": {
-        "find_connection": ["find",
+        "find_connection": ["find connection", "find a connection", "find out connection", "find link", "find a link"
```

Comment:
"Find out", "link" are IMHO not appropriate for this context. I'd add "give (me) directions".